modules/cluster-logging-loki-deploy.adoc

```diff
@@ -0,0 +1,132 @@
+// Module is included in the following assemblies:
+//cluster-logging-loki.adoc
+:_content-type: PROCEDURE
+[id="logging-loki-deploy_{context}"]
+= Deploying the LokiStack
+You can use the {product-title} web console to install the LokiOperator.
+
+.Prerequisites
+
+* {logging-title-uc} Operator 5.5
+* S3 bucket for log storage
+
+To install the LokiStack using the {product-title} web console:
+
+. Install the LokiOperator:
+
+.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+.. Choose  *LokiOperator* from the list of available Operators, and click *Install*.
+
+.. Under *Installation Mode*, select *All namespaces on the cluster*.
+
+.. Under *Installed Namespace*, select *openshift-operators-redhat*.
++
+You must specify the `openshift-operators-redhat` namespace. The `openshift-operators`
+namespace might contain Community Operators, which are untrusted and could publish
+a metric with the same name as an {product-title} metric, which would cause
+conflicts.
+
+.. Select *Enable operator recommended cluster monitoring on this namespace*.
++
+This option sets the `openshift.io/cluster-monitoring: "true"` label in the Namespace object.
+You must select this option to ensure that cluster monitoring
+scrapes the `openshift-operators-redhat` namespace.
+
+.. Select an *Approval Strategy*.
++
+* The *Automatic* strategy allows Operator Lifecycle Manager (OLM) to automatically update the Operator when a new version is available.
++
+* The *Manual* strategy requires a user with appropriate credentials to approve the Operator update.
+
+.. Click *Install*.
+
+.. Verify that you installed the LokiOperator. Visit the *Operators* → *Installed Operators* page and look for "LokiOperator."
+
+.. Ensure that *LokiOperator* is listed with *Status* as *Succeeded* in all the projects.
++
+. Create a Secret YAML file that uses the aws_access_key_id and aws_secret_access_key fields to specify your base64-encoded AWS credentials. For example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: logging-loki-s3
+  namespace: openshift-logging
+data:
+  aws_access_key_id: QUtJQUlPU0ZPRE5ON0VYQU1QTEUK
+  aws_secret_access_key: d0phbHJYVXRuRkVNSS9LN01ERU5HL2JQeFJmaUNZRVhBTVBMRUtFWQo=
```

Comment:
This configuration is not sufficient, the secret also needs to contain the bucket name, endpoint and optionally region:

```suggestion
  access_key_id: QUtJQUlPU0ZPRE5ON0VYQU1QTEUK
  access_key_secret: d0phbHJYVXRuRkVNSS9LN01ERU5HL2JQeFJmaUNZRVhBTVBMRUtFWQo=
  bucketnames: czMtYnVja2V0LW5hbWU= # base64-encoded bucket name
  endpoint: aHR0cHM6Ly9zMy5ldS1jZW50cmFsLTEuYW1hem9uYXdzLmNvbQ== # base64-encoded S3 API endpoint
```